pipelines/toast_satellite_sim.py

```diff
@@ -124,6 +138,45 @@ def simulate_sky_signal(args, comm, data, mem_counter, focalplanes, subnpix, loc
 
     mem_counter.exec(data)
 
+def simulate_sky_signal_conviqt(args, comm, data, mem_counter, focalplanes,signalname):
+    # Convolve a signal TOD from PySM
+    start = MPI.Wtime()
+
+    # Prepare the detectordata list
+
+    local_dets = extract_local_dets(data)
+
+    for pattern in args.conviqt_beamfile_alm.split(','):
+        detectordata = []
+        for det in local_dets:
+            skyfile = args.conviqt_input_alm.replace('DETECTOR', det)
+            beamfile = pattern.replace('DETECTOR', det)
+            # polarization leakage
+            epsilon = 0.
+            psipol = 0. if det.endswith("A") else np.pi/2
```

Comment:
@keskitalo assuming `psipol` 90 deg is just temporary. should we add this to the focalplane pkl?

Reply:
or computing it from the detector quaternion?